cards/src/main/java/com/decathlon/vitamin/compose/cards/VitaminCards.kt

```diff
@@ -0,0 +1,258 @@
+package com.decathlon.vitamin.compose.cards
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawWithCache
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.unit.dp
+import com.decathlon.vitamin.compose.buttons.VitaminButtons
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+import com.decathlon.vitamin.compose.foundation.vtmnLightColorPalette
+
+object VitaminCards {
```

Comment:
Need to be applied on all components here:

Reply:
All padding inside components should be refactored inside a `VitaminCardSizes` object and passed in parameter to be overridden by the consumer.

Reply:
I'm wondering if we should add an empty `VitaminCard` component and used in all variants here. What do you think?